mono/metadata/monitor.c

```diff
@@ -791,6 +791,10 @@ signal_monitor (gpointer mon_untyped)
 	mono_coop_mutex_unlock (mon->entry_mutex);
 }
 
+#if ENABLE_NETCORE
+static gint32 thread_contentions; /* for Monitor.LockContentionCount, otherwise mono_perfcounters struct is used */
```

Comment:
looks like this should be gint64

Reply:
Yea, so @BrzVlad brings up a good point.  The icall is defined to return a 64-bit value for the contention count.  Mono's old perfcounter code used a 32-bit counter, however.
I think for netcore we should use a 64-bit counter here, @alexischr